lib/src/components/NxVulnerabilityDetails/NxVulnerabilityDetails.scss

```diff
@@ -131,13 +111,32 @@
     }
 
     blockquote {
-      border-left: 5px solid $nx-grey-2;
+      @include container-vertical();
+      background-color: #f4f5f9;
+      border: 1px solid #d1d6f0;
+      border-left-width: 5px;
+      border-radius: $nx-border-radius;
       margin: 0 0 20px;
-      padding: 0 0 0 15px;
+      padding: $nx-spacing-l $nx-spacing-l $nx-spacing-l $nx-spacing-xl;
+    }
+
+    // per design make these look like an H4
```

Comment:
I don't agree with this at all.  We have no guarantee that all `<em>` tags will be used in this way.  When the data researchers write `*stuff*`, their intent is italics.  We should honor that intent not try to guess what deeper meaning they had for that element.

Reply:
> I don't agree with this at all. We have no guarantee that all <em> tags will be used in this way. When the data researchers write *stuff*, their intent is italics. We should honor that intent not try to guess what deeper meaning they had for that element.

I also no longer see the relevant elements in the mockup.

Reply:
> I also no longer see the relevant elements in the mockup.

Additionally, I think the content of our example here was misinterpreted.  `*Component Specific Explanation.*` was not something that was present and intended as a header in the original JSON data from the data team.  Rather, it was something _we_ added to the example to give it some "component-specific" content.  "Component Specific Explanation." is not meant to be taken literally as a header here; it is intended to be a placeholder for some actual component-specific explanatory text, which may or may not include `*`s (probably not) in a real example from the data team.

Reply:
> I don't agree with this at all. We have no guarantee that all `<em>` tags will be used in this way. When the data researchers write `*stuff*`, their intent is italics. We should honor that intent not try to guess what deeper meaning they had for that element.

Ok, restored here: https://github.com/sonatype/sonatype-react-shared-components/pull/129/commits/06b50da56a3aaf5844f3cffe27527768d7c4bc24